test/dialect/mssql/test_compiler.py

```diff
@@ -1474,9 +1474,7 @@ def test_index_include_where(self):
         )
 
     def test_try_cast(self):
```

Comment:
**Nick Crews** (NickCrews) wrote:

Ack

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4605